FlowCrypt/Functionality/Pgp/KeyMethods.swift

```diff
@@ -63,8 +75,46 @@ final class KeyMethods: KeyMethodsType {
             armoredOrBinary: armored.joined(separator: "\n").data()
         )
         guard parsed.keyDetails.count == armored.count else {
-            throw KeyServiceError.parsingError
+            throw KeyMethodsError.parsingError
         }
         return parsed.keyDetails
     }
+
+    func chooseSenderSigningKey(keys: [Keypair], senderEmail: String) async throws -> Keypair? {
+        // TODO - needs to be implemented
+        return keys.first
+    }
+
+    func chooseSenderEncryptionKeys(keys: [Keypair], senderEmail: String) async throws -> [Keypair] {
+        // TODO - needs to be implemented
+        return keys
+    }
```

Comment:
Refactoring is more or less done. Now I need to actually implement these two methods.

This should make it a lot clearer what is going on.